.babelrc

```diff
@@ -25,6 +25,11 @@
           }
         ]
       ]
+    },
+    "production": {
+      "plugins": [
+        ["transform-remove-console", { "exclude": ["error", "warn"] }]
```

Comment:
can use full form of plugin name?
want to be able to grep the package name and then know where/do we use it

Reply:
sure